cli/tools/upgrade.rs

```diff
@@ -864,13 +1035,14 @@ struct CheckVersionFile {
   pub last_checked: chrono::DateTime<chrono::Utc>,
   pub current_version: String,
   pub latest_version: String,
+  pub current_release_channel: ReleaseChannel,
```

Comment:
I opted to store this info in the `latest.txt` file so we don't have to hit remote server to display a banner. Works quite well 👍